docs/src/main/sphinx/release/release-460.md

```diff
@@ -0,0 +1,51 @@
+# Release 460 (2 Oct 2024)
+
+## Atop connector
+
+* {{breaking}} Remove the Atop connector. ({issue}`23550`)
+
+## ClickHouse connector
+
+* Improve performance of listing columns. ({issue}`23429`)
+* Improve performance for queries comparing `varchar` columns. ({issue}`23558`) 
+* Improve performance for queries using `varchar` columns for `IN` comparisons. ({issue}`23581`)
+* Improve performance for queries with complex expression for `LIKE` clauses. ({issue}`23591`)
+
+## Delta Lake connector
+
+* Add support for using an [Alluxio cluster as file system
+  cache](/object-storage/file-system-alluxio). ({issue}`21603`)
+* Fix writing of bloom filter columns in Parquet files. ({issue}`22701`)
+
+## Hive connector
+
+* Add support for using an [Alluxio cluster as file system
+  cache](/object-storage/file-system-alluxio). ({issue}`21603`)
+* Fix writing of bloom filter columns in Parquet files. ({issue}`22701`)
+
+## Hudi connector
+
+* Fix writing of bloom filter columns in Parquet files. ({issue}`22701`)
```

Comment:
i forgot we don't have writes in hudi, please skip this one

Reply:
Haha.. me too. Its time we get that update to Hudi 1.0 and the related improvements and more features ;-)